src/DetailsView/components/start-over-dropdown.tsx

```diff
@@ -21,6 +23,7 @@ export interface StartOverProps {
     actionMessageCreator: DetailsViewActionMessageCreator;
     test: VisualizationType;
     requirementKey: string;
+    detailsViewRightContentPanel: DetailsViewRightContentPanelType;
```

Comment:
Instead of adding `detailsViewRightContentPanel` consider making `testName` and `test` optional. Then this component doesn't need to know the rules around when to show the test item - it just doesn't show it if `test` is not provided. The responsibility for deciding whether to have the item can lie elsewhere.

Reply:
I was thinking doing that as well. Problem being we always have selected test, it never become null.